Core/Locales/common.itIT.lua

```diff
@@ -130,8 +130,8 @@ L.link_with_rticon = "{rt%d}Vincolato a %s"
 L.link_both = "%s vincolato a %s" -- XXX needs updated
 L.link_both_icon = "|T13700%d:0|t%s vincolato a |T13700%d:0|t%s" -- XXX needs updated
```

Comment:
Please update both of these as the English version has changed.

Reply:
Hi, where can I find the english version to use as a reference ?